server/api/controllers/posts.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const router = express.Router();
+
+//const Dog = require('../models/dog')
+const Post = require('../models/post')
+
+
+// dogs index route
+router.get('/', async (req, res) => {
+    try {
+        const posts = await Post.all
+        res.json({posts})
```

Comment:
I havent actually looked at the client side but not sure you need to return posts within an object. I think this sends it as a key rather than the values. I could be wrong